test/performance/alphabet/alphabet_assign_char_benchmark.cpp

```diff
@@ -0,0 +1,64 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#include <algorithm>
+#include <cstring>
+
+#include <benchmark/benchmark.h>
+
+#include <seqan3/alphabet/all.hpp>
+
+using namespace seqan3;
```

Comment:
Mhm, don't we want to get rid of it in _every_ test?

Reply:
Well, I don't know. My assumption was:
* unit test ensure that our interfaces can be used in the advertised way and that we have code coverage. Since we advertise using all things qualified, that's what we should test.
* performance tests highlight performance issues and regressions. I don't see how calling qualified/unqualified has an impact on this, so I left it this way.

Reply:
So everywhere **except** in performance tests.
Then we should write this down more clearly somewhere.

Reply:
Yes. I mean I don't really care. We can also say that we do it everywhere for consistency.

Reply:
Ok, I have made everything qualified.